tests/unit/commands/init/test_cli.py

```diff
@@ -230,7 +231,7 @@ def test_init_cli_int_from_location(self, generate_project_patch, sd_mock):
     def test_init_cli_missing_params_fails(self):
         # WHEN we call init without necessary parameters
         # THEN we should receive a UserException
-        with self.assertRaises(UserException):
+        with self.assertRaises(click.UsageError):
```

Comment:
Something to note, usage errors are exit code 2. Would we be breaking anyone?

Reply:
This is only for init and the all the errors are based on either passing in the wrong data (not json) or invalid collection of parameters. Given it is only an exit code and for init, I am not concerned with this breaking anyone.